pandas/core/series.py

```diff
@@ -3177,7 +3177,8 @@ def _try_cast(arr, take_fast_path):
 
     # GH #846
     if isinstance(data, (np.ndarray, Index, Series)):
-
+        if data.ndim > 1:
```

Comment:
actually should error on data.ndim != 1 (e.g. a numpy scalar of ndim==0 I think hits this path maybe)

Reply:
can you add tests to cover this case

Reply:
Shouldn't we allow numpy scalars? (just like normal scalars work, eg like `Series(1)`)

Reply:
Sure we do! And this is widely tested (as opposed to the initialization with length 1 list-likes, which was working by accident, accidentally used in the tests, and I now disabled).